src/components/utils/src/threads/thread_delegate_win.cc

```diff
@@ -29,7 +29,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-#if defined(WIN_NATIVE)
+#if defined(OS_WINDOWS)
```

Comment:
All this stuff with OS_WINDOWS will be removed from specific/system-dependent place.
Right now it needs because of compilation problem.
I.e threads on QT should be ported in separate task.
Then define from this class will be splited or removed or some thing else